addons/escoria-core/game/core-scripts/esc_item.gd

```diff
@@ -302,6 +295,25 @@ func _unhandled_input(event: InputEvent) -> void:
 				get_tree().set_input_as_handled()
 
 
+func _is_in_shape(position: Vector2) -> bool:
+	var colliders = get_world_2d().direct_space_state.intersect_point(
+		position,
+		32,
+		[],
+		2147483647,
+		true,
+		true
+	)
+	for _owner in get_shape_owners():
```

Comment:
Hmm, from https://docs.godotengine.org/en/stable/classes/class_collisionobject.html:

> CollisionObject is the base class for physics objects. It can hold any number of collision [Shape](https://docs.godotengine.org/en/stable/classes/class_shape.html#class-shape)s. Each shape must be assigned to a shape owner. The CollisionObject can have any number of shape owners.

so for a 2D adventure game, I'm guessing these numbers are generally small?

Reply:
I sure hope so. :) If there's no other way to accomplish this and as long as @dploeger and @StraToN are good with it, then I'm good.

WDYT, folks?

Reply:
I already accepted the PR. 😄

Reply:
Just wanted to make sure you saw this convo. :P

Will resolve this now.